main.py

```diff
@@ -1,3 +1,2 @@
-a =('Hello')
 
-print(a)
+print('Hello world'
```

Comment:
Nu este paranteza la final
```suggestion
print('Hello world')
```